### PR TITLE
[FAQ] Reorganize FAQ and add tools and pipelines questions

### PR DESCRIPTION
**Pull request by cmadjar**
This PR does the following:
- reorganization of the FAQ into 5 sections: 
  - CONP, 
  - Data Access & Download, 
  - Data Upload, 
  - Tools & Pipelines, 
  - DataLad
- addition of tools and pipelines questions to the FAQ

This solves #16

### CODE REVIEW

**Comment by cmadjar** [comment]
@glatard could you take a look at the added pipelines questions to the FAQ to ensure they make sense?

@emmetaobrien could you ensure the English is correct? I may have inserted a few typos.

Would be good to have that merged before the presentation to Alan tomorrow in case we want to show him the FAQ.

Thank you both!!

File: CONP_FAQ.md [hunk 1]
```diff
@@ -1,47 +1,137 @@
...
+CONP pipelines can be easily installed and run on any computer with a container engine. You will need to install:
+
+- [Docker](https://www.docker.com/) or [Singularity](https://singularity.lbl.gov/) 
+- [Boutiques](https://pypi.org/project/boutiques/) python package (`pip install boutiques`)
+
+#### **How can I run a CONP tool/pipeline?**
+
+First, ensure your have [Boutiques](https://pypi.org/project/boutiques/) installed with [Docker](https://www.docker.com/) or [Singularity](https://singularity.lbl.gov/).
+
+If you have never used Boutiques in the past, ensure to go through the following [tutorial](https://nbviewer.jupyter.org/github/boutiques/tutorial/blob/master/notebooks/boutiques-tutorial.ipynb#reusing_tools).
```
Comment (emmetaobrien) [inline CONP_FAQ.md]: "ensure to go through" is awkward, I would suggest "we recommend the following tutorial".

File: CONP_FAQ.md [hunk 2]
```diff
@@ -1,47 +1,137 @@
...
+    run('bids-validator %s'%args.bids_dir)
+  File "/run.py", line 25, in run
+    raise Exception("Non zero return code: %d"%process.returncode)
+Exception: Non zero return code: 2
+...
+Exit code
+1
+```
+
+=> ensure to mount the directory where your data is on the container by using the `-v` option in the bosh command. For example, if your data is in `/data`, then you need to specify `-v /data:/data` in the `bosh` command as follows:`bosh exec launch -s <zenodo_id> <invocation> -v <directory_to_mount>`
```
Comment (emmetaobrien) [inline CONP_FAQ.md]: "ensure to mount the directory where your data is"->"confirm your data directory is mounted"

File: CONP_FAQ.md [hunk 3]
```diff
@@ -1,47 +1,137 @@
...
+
+#### **How can I upload/share my own tool/pipeline on CONP?**
+
+You can add your own tool/pipeline by following the instructions described in this [document](https://nbviewer.jupyter.org/github/boutiques/tutorial/blob/master/notebooks/boutiques-tutorial.ipynb#publishing_tools).
+
+## <a name="datalad"></a> DataLad
+
+#### **To which username/password is DataLad referring to when trying to download via `datalad get` preventad-open or multicenter-phantom datasets?**
 
 In order to have access to those datasets, you will need to request an account on their respective LORIS repository.
```
Comment (emmetaobrien) [inline CONP_FAQ.md]: "repositories" I think.

**Pull request merged**